dbt/parsely_dpl/historical_script.py

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
Nitpick: new lines and missing python 3 compat imports

* I think a docstring would also be helpful for each of these scripts, and perhaps a doc explaining in more detail how they compare to each other and/or are used together.